contracts/gov/GovPool.sol

```diff
@@ -62,7 +62,7 @@ contract GovPool is
 
     uint256 internal _latestProposalId;
```

Comment:
Please make this parameter `public`, otherwise there is no way to know how many proposals to paginate against